src/scss/skin-modern/_mixins.scss

```diff
@@ -144,3 +144,19 @@
     outline: none;
   }
 }
+
+/// Replace `$search` with `$replace` in `$string`. From https://css-tricks.com/snippets/sass/str-replace-function/
+/// @author Kitty Giraudel
```

Comment:
Do we want to keep this?

Reply:
I thought it would be nice to keep the attribution. But the link alone could also be sufficient. What's your preference?

Reply:
I don't have a strong opinion. Both works for me 👍

Reply:
Updated in 8c5bab4d7413435849d711f5822c42ea1c895383